src/Components/QuestionList/QuestionViewTypes/QuestionListTable/QuestionListTableS.scss

```diff
@@ -0,0 +1,14 @@
+.QuestionListTable {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 10rem;
```

Comment:
remove, and don't dod this fixed stuff. it was testing so it should have been removed and not pushed. but should not test like this to begin with.